github-adapter/src/main/java/be/xplore/githubmetrics/githubadapter/JobsAdapter.java

```diff
@@ -12,32 +13,32 @@
 import java.util.List;
 
 @Component
-public class WorkflowRunJobsAdapter implements WorkFlowRunJobsQueryPort {
-    private static final Logger LOGGER = LoggerFactory.getLogger(WorkflowRunJobsAdapter.class);
+public class JobsAdapter implements JobsQueryPort {
```

Comment:
wonder if the Adapters should be named more specifically like we are doing for the exporter

Reply:
I agree but i think doing this in the big refactor when everythings place has been finalized will make this easier